jwst/regtest/test_miri_image.py

```diff
@@ -66,10 +66,12 @@ def run_pipelines(jail, rtdata_module):
 
 
 @pytest.mark.bigdata
-@pytest.mark.parametrize("suffix", ["a3001_crf", "assign_wcs", "cal", "dark_current",
-    "dq_init", "firstframe", "flat_field", "i2d", "lastframe", "linearity", "ramp",
-    "rate", "rateints", "refpix", "rscd", "saturation"])
-def test_miri_image_stages12(run_pipelines, fitsdiff_default_kwargs, suffix):
+@pytest.mark.parametrize("suffix", ["dq_init", "saturation", "refpix", "rscd",
+    "firstframe", "lastframe", "linearity", "dark_current", "ramp", "rate",
+    "rateints",
+    "assign_wcs", "flat_field", "cal", "i2d",
+    "a3001_crf"])
```

Comment:
Just curious, is this ordering necessary so that it first tests all the results from ``detector1``, then the results from ``image2``, and finally the results from ``image3``? Otherwise it would be doing comparisons of files that hadn't been regenerated yet?

Reply:
No, not necessary, just a little clearer to me the reader as to what is getting tested.  I was confused as to why `a3001_crf` was getting tested at all, but then realized it was the stage3 processing that was producing this (and due to the non-standard naming) putting the asn id at the end.